composer.json

```diff
@@ -12,8 +12,8 @@
     }
   },
   "require": {
-    "shopware/core": "~6.4.0",
-    "shopware/storefront":  "~6.4.0"
+    "shopware/core": "6.5.x-dev",
+    "shopware/storefront":  "6.5.x-dev"
```

Comment:
Shouldn't this refer to a stable version and not the main/dev branch?

Reply:
The stable version is NOT available yet, only the RC version is available now.
Maybe we will wait until the stable is available.

Reply:
Okay, I would say for merging we should wait, yes.

Reply:
hey guys! I think we are ready to set the dependency for `^6.5.0.0` now 🚀 
I will proceed with the topic in order to make the PWA working well with the latest release as well -> https://github.com/vuestorefront/shopware-pwa/issues/1992

Reply:
@mkucmus I updated it with the official 6.5.0.0.